app/assets/builds/alchemy/welcome.css

```diff
@@ -0,0 +1 @@
ZINOpqpLOzJT2Qzwc7O0PhutUVVFZIWSNXI8rqUlZUWTFlxZWVqFtRRYsGhW+oM3VWMKLiBlW0oIoWVNFCI7mrjEMq45DKM6TihVSeIRU5pCKHVOSwihxWkcMqclj1RVgxwooRVoywYoQVI6wYEcWIKEZEMSKKEVGMkX6JKEZEMSKKERnpb9UiqlpEVYuoahFVLWIqq5jKJaZyialcYipyTEWOqcgxFTmmIsdV5LiqN64YccWIK0ZcMeKKEVeMuGLEFSOhGAnFSChGQjESipFQjIRiJGLe3Owa6YuRaHVWUK0c+5+y1BzuDCsroqwuZUWVFVNWXFkJ4dfXRjCoWAnvFyIS7RSVdRQNidavR0S7RGW9R6W2aFxUWNGkqCaqixqipmjK05jwY8KPCT8m/JjwY8KPCT8m/JjwY8KPCT8m/JjwY8KPCT8m/Ljw48KPCz8u/Ljw48KPCz8u3Lhw48KNCzcu3Lhw48KNp9TTbGSOSayEsOtjkxB2QtgJYdfnlWonOSQkh4TkkJAcEpJDQnJISO1JqT0ptSeFnxR+UvhJ4SeFnxR+UvhJ4SeFnxR+UvhJ4SeFnxS+JnxN+JrwNeFrwteErwlfE74mfE34mvA14cvbUEQTviZ8Tfi68HXh68LXha8LXxe+Lnxd+LrwdeHrwteFrwtfF74ufF34hvDlNzUiv6kRQ/iG8A3h139rDeEbwjeEbwjfEL4hfEP4hvAN4ZvCN4VvCt8Uvil8U/im8E3hm8I3hW8K3xS+KXxT+KbwTeGnhJ8Sfkr48vYaSQk/JfyU8FPCTwk/JfyU8FPCTwk/JfyU8FMev0vejrvk7birMyQaFo2IdolGRWOicdGEaFJUE9VFDVFTVPhB4QeFHxR+0OUHTS8/W4Oehk3RlH9utpy2P5SGPZnrfcAMu9Iyt/5a1zJct7x2SYmXlHjeerdVeN56t7VLNCoaE42LJkSTopqoLmqISr7eeg+amvA14WvC14SvCV8TviZ8Tfia8DXha8LXhK8JXxO+JnxN+LrwdeHrwteFrwtfF74ufF34uvB14evC14WvC18Xvi58XfiG8A3hG8I3hG8I3xC+IXxD+IbwDeEbwjeEbwjfEL4hfEP4pvBN4ZvCN4VvCt8Uvil8U/im8E3hm8I3hW8K3xS+KXxT+Cnhp4L++d7EXeqKnBV6SugpoaeEnhK6t9qDqYhHSXlVhjq91WJrUDQkGhaNiHaJRkVjonHRhGhSVPM0KPGDwdaBfLZWtvrtD3f3J9z5goh31Ir9sonX31fo2LtmfzE7uxjlitXvNQyl/IP5orubYmXsj68Wa0nG/sKz7/aud6Xal1nlUqU2ZJXzpXL7QKlWHnHyi62601bJL6nbHRWbUVSelc/mqqpRMV9UjVrUTV7O8UQk6B+yKvYnp4dPak1mrVxyr0aDIfkMsC15LY4Gw11elwbtDyarUs0PpqtWf0upaLnUXHs1V7bErrQ5CYvd7uYojhshIoPVldJpYMAxU5o3braGRMOi7rilTO+pnUp5T3VbDV/3xHSh2trtTiLPdJ+Ajjm6e2SDx/E7uuvPQu9Gd//JMdtkO9E77W48eae9zSTHHtWt9oEc1989xYtY/+p3vGbHc4wxjVsJ7n3qK3cFL+p6maXlfKGQz7jcMe5mWePWaIM9tsEuO2SrTV5i3HTlxaJuOw+rBjvp2ONHesTtJjtgX8M5txedc+NGzjmbRf+6ze0459woN2cVyXNVkA7Xrbf3LqqmY+plO9cL1oBdXL0f7Elin+vV/31mmt5sD8fEoWJtMOCskomlit85YUt12PGa3dlnGwFn3bhqz0Dnur1anOvuJHQMdyI6NzjLw1Zf1p0w2RUnTGtWzYVmZwPbHdvGfWvnxP8BVFAtHHjaY/DewXAiKGIjI2Nf5AbGnRwMHAzJBRsZ2J22MQQ6GiqyMmiBOA48YUxuHGYcahwS7KwcUKEIBj82OzY9NnlWsBCP0z7RA4IHeA9wHmBzYGBl4ASKCTrtY3CAQ7AYM4PLRhXGjsCIDQ4dERuZU1w2qoF4uzgaGBhZHDqSQyJASiKBAGSDB5sFmwabFCsrj9YOxv+tG1h6NzIxuGxmTWFjcHEBAAb8LMkAAAAAAVqoVgAAAA==) format("woff");font-weight:700;font-style:normal}body{background-color:var(--color-blue_dark);color:var(--color-white);font-family:"Open Sans","Lucida Grande","Lucida Sans Unicode","Lucida Sans",Verdana,Tahoma,sans-serif;font-weight:normal;font-size:15px;line-height:22px}h3{font-weight:bold;font-size:25px;line-height:1.2;margin-bottom:20px;padding-top:20px}.container{position:absolute;top:50%;left:50%;width:600px;margin-top:-50px;transform:translateY(-50%) translateX(-300px)}ul{line-height:1.6;margin-bottom:17px;padding:0;padding-left:20px;margin:0px;list-style-position:outside}ul li{color:var(--color-white);margin-bottom:12px;background-color:none;border-bottom:0 none;line-height:22px}a{color:var(--color-white);text-decoration:underline;transition:all .1s ease-in-out}a:hover{color:var(--color-white);text-decoration:none}/*# sourceMappingURL=welcome.css.map */
```

Comment:
Idea: Split up custom properties into smaller pieces (`colors`, etc.) in order to keep this file small